src/03-modality-agnostic-files.md

```diff
@@ -304,3 +304,299 @@ code organization of these scripts at the moment.
 <sup>1</sup>Storing actual source files with the data is preferred over links to
 external source repositories to maximize long term preservation (which would
 suffer if an external repository would not be available anymore).
+
+## Provenance of BIDS datasets, files, and derivatives
+
+Template:
+
+```Text
+- [Dataset level] prov.jsonld
+- [File level] sub-<label>/[ses-<label>/]sub-<label>[_ses-<label>]_prov.jsonld
```

Comment:
```suggestion
- [File level] sub-<label>/[ses-<label>/]sub-<label>[_ses-<label>]_<suffix>.prov
```
To reflect @effigies suggestion.

Reply:
We just discussed this at OHBM Brainhack 2021 and agreed on using `_<suffix>.jsonld`